### PR TITLE
Fix forward declarations for NSArray and NSDictionary in UIPasteboard.h

### PR DESCRIPTION
**Pull request by asimonov-msft**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1433)

### CODE REVIEW

**Comment by DHowett-MSFT** [comment]
:shipit:

**Comment by DHowett-MSFT** [comment]
I do wonder if it is better to simply include the necessary headers. After all, you _cannot actualy use_ UIPasteboard without having imported the rest of the foundation headers, no matter how hard you try.

**Comment by pradipd** [comment]
:shipit:

**Comment by oliversa-msft** [comment]
Agreed

---
In reply to: [262001903](https://github.com/Microsoft/WinObjC/pull/1433#issuecomment-262001903) [](ancestors = 262001903)

File: include/UIKit/UIPasteboard.h [hunk 1]
```diff
@@ -20,7 +20,8 @@
 #import <Foundation/NSObject.h>
 
 @class NSString;
-@class NSArray;
+@class NSArray<ObjectType>;
```
Comment (yiyang-msft) [inline include/UIKit/UIPasteboard.h]: >ObjectType [](start = 15, length = 10)

what is the difference between these two usage?

**Comment by oliversa-msft** [comment]
:shipit:

**Comment by yiyang-msft** [comment]
:shipit:

File: include/UIKit/UIPasteboard.h (hunk 1 above)
Comment (asimonov-msft) [inline include/UIKit/UIPasteboard.h]: NSArray<ObjectType>is how you forward-declare a generic type.

---
In reply to: [88955275](https://github.com/Microsoft/WinObjC/pull/1433#discussion_r88955275) [](ancestors = 88955275)

**Pull request merged**